R/api.R

```diff
@@ -307,6 +307,37 @@ endpoint_kill <- function(runner) {
     returning = returning_json("Kill.schema"))
 }
 
+target_dependencies <- function(path, name,
+                                id ="latest",
```

Comment:
```suggestion
                                id = "latest",
```

Reply:
Surprised the linter didn't pick this up, it moaned about everything else! :laughing: